### PR TITLE
[TT-10177] Update documentation for master

### PR DESCRIPTION
**Pull request by buger**
Triggered by: lghiur

Included:

Tyk Gateway: true
Tyk Dashboard: true
Tyk MDCB false
Tyk Pump false

Intended for: master

Note: Docs updates for 5.2.1

JIRA: https://tyktech.atlassian.net/browse/TT-10177

### CODE REVIEW

File: tyk-docs/assets/others/dashboard-swagger.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Tyk Dashboard API
-  version: 5.2.0
+  version: 5.0.0
```
Comment (dcs3spp) [inline tyk-docs/assets/others/dashboard-swagger.yml]: If no patched API changes then:

```suggestion
  version: 5.2.0
```

File: tyk-docs/assets/others/gateway-swagger.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Tyk Gateway API
-  version: 5.2.0
+  version: 5.1.0
```
Comment (dcs3spp) [inline tyk-docs/assets/others/gateway-swagger.yml]: If no patched API changes then:

```suggestion
  version: 5.2.0
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
lgtm, minor change suggested relating to the version used on swagger files

**Review by dcs3spp — APPROVED** [review]

File: tyk-docs/content/shared/gateway-config.md [hunk 3]
```diff
@@ -1902,3 +1909,18 @@ Type: `bool`<br />
 
 Skip TLS verification for JWT JWKs url validation
 
+### resource_sync
```
Comment (letzya) [inline tyk-docs/content/shared/gateway-config.md:1912]: @dcs3spp why do we have only a description?

Comment (dcs3spp) [reply tyk-docs/content/shared/gateway-config.md:1912]: @letzya This a description for the config entity. The sentence is an overview for what the entity represents.

Then the items below are the attributes of that entity, e.g. resource_sync.retry_attempts and resource_sync.interval

**Pull request merged**